### PR TITLE
Misc doc changes

### PR DESCRIPTION
**Pull request by kianmeng**
Besides other documentation changes, this commit ensures the generated
HTML doc for HexDocs.pm will become the source of truth for this Elixir
library and leverage on latest features of ExDoc.

### CODE REVIEW

**Comment by kianmeng** [comment]
@josevalim Done. Please check, I'm assumed this project is maintained by Dashbit after ownership transfer?

File: lib/tds/binary_utils.ex [hunk 1]
```diff
@@ -120,52 +141,56 @@ defmodule Tds.BinaryUtils do
...
 
   @doc """
-  A 16-bit signed integer
+  A 16-bit signed integer.
   """
   defmacro int16(), do: quote(do: signed - 16)
 
   @doc """
-  A 16-bit signed integer
+  A 16-bit signed integer.
```
Comment (moogle19) [inline lib/tds/binary_utils.ex:163]: This should be `A 32-bit signed integer.`.
The following documentation is also wrong. 
Would be nice if you could fix this.

**Review by moogle19 — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -212,15 +213,14 @@ Thanks to [ericmj](https://github.com/ericmj), this driver takes a lot of inspir
 
 Also thanks to everyone in the Elixir Google group and on the Elixir IRC Channel.
 
-## License
+## Copyright and License
 
-Copyright 2014, 2015, 2017 LiveHelpNow
+Copyright (c) 2015 LiveHelpNow \
+Copyright (c) 2021 Dashbit
```
[the remark this answers is not in the document]
Comment (kianmeng) [reply README.md:219]: > I am not sure if you should add Dashbit here. But I also have no idea about Copyright.

@josevalim Is `:tds` library under maintenance of `Dashbit` or someone else right now?

[the remark this answers is not in the document]
Comment (josevalim) [reply README.md:219]: It is not. Generally speaking, don’t change copyright lines, it is really up to maintainers. :)

[the remark this answers is not in the document]
Comment (josevalim) [reply README.md:219]: To be clear, @moogle19 is maintaining it right now and they can add their own copyright, but I believe he doesn’t have to either. :)

[the remark this answers is not in the document]
Comment (kianmeng) [reply README.md]: @josevalim Noted. Extra copyright line removed.
@moogle19 Please check again.

**Comment by kianmeng** [comment]
@moogle19 Should I resolve the conflict and resubmit the PR again?

**Comment by moogle19** [comment]
@kianmeng 
Sorry for the late reply. Yes, that would be awesome!

**Comment by kianmeng** [comment]
@moogle19 Done. Please check again.

**Comment by moogle19** [comment]
@kianmeng 
Thanks a lot!

**Comment by kianmeng** [comment]
:+1: :tada:

**Pull request merged**